src/scripts/main.js

```diff
@@ -1,3 +1,32 @@
 'use strict';
 
-// write your code here
+function thousandsSeparator(number) {
+  const splitedNum = String(number).split('');
+  const modificatedNum = [];
+  let counter = 0;
+
+  for (let i = splitedNum.length - 1; i >= 0; i--) {
+    counter++;
+    modificatedNum.push(splitedNum[i]);
+
+    if (counter % 3 === 0 && i !== 0) {
+      modificatedNum.push(',');
+    }
+  }
+
+  return modificatedNum.reverse().join('');
+}
+
+const populationsList = document.querySelectorAll('.population');
```

Comment:
if we know that we'll have only one list, will be better to use simple `querySelector`